website/package-lock.json

```diff
@@ -3913,8 +3913,7 @@
             "ansi-regex": {
               "version": "2.1.1",
               "bundled": true,
-              "dev": true,
```

Comment:
just curious why all of these were marked as optional before, but not needed any more

Reply:
Oops, this is not supposed to be added. Reverted all changed in this file.